projects/packages/connection/src/class-manager.php

```diff
@@ -161,32 +160,27 @@ public static function configure() {
 	 * Sets up the XMLRPC request handlers.
 	 *
 	 * @since 1.25.0 Deprecate $is_active param.
+	 * @since $$next-version$$ Remove $request_params param.
 	 *
-	 * @param array                 $request_params incoming request parameters.
 	 * @param bool                  $has_connected_owner Whether the site has a connected owner.
 	 * @param bool                  $is_signed whether the signature check has been successful.
 	 * @param Jetpack_XMLRPC_Server $xmlrpc_server (optional) an instance of the server to use instead of instantiating a new one.
 	 */
 	public function setup_xmlrpc_handlers(
-		$request_params,
```

Comment:
Sorry if I'm being too nitpicky here, but can I suggest we go with something like this:
```
public function setup_xmlrpc_handlers( $deprecated = null, $whatever_args_go_next ) {
    if ( ! is_null( $deprecated ) ) {
        _deprecated_argument('$request_params', '$$next-version$$', 'Passing $request_params to setup_xmlrpc_handlers' );
    }
 ```

Reply:
Nice suggestion, thanks!
I went with:
```
_deprecated_argument( __METHOD__, '$$next-version$$' );
```

the first argument in `_deprecated_argument` is the method. Given that, I think the message might be redundant :)

Also I'd avoid setting it to `null` by default because there are more required arguments after that and optional arguments should go after the required ones.